Dockerfile

```diff
@@ -7,6 +7,7 @@ WORKDIR /app
 COPY go.mod go.sum ./
 RUN go mod download
 COPY cmd/backup ./cmd/backup/
+COPY internal ./internal
```

Comment:
Can we copy these in the same step so there's still only a single Docker layer created for the source code?

Reply:
COPY only supports multiple files or one directory incl. files, not multiple directories (while also copying the directory tree structure with it).
Since it's inside the builder, the common approach would be to copy everything in there, so the complete folder structure. The `go build` only uses the go files anyway and compiles to a single binary, which is copied to the actual image. So anything we do in the builder, doesn't really matter. We could still remove some directories (instead of using everything) by using .dockerignore for that.

Reply:
True. I do like the idea of having a `.dockerignore` file that ignores everything but relevant source files. If you want to add that, it'd be cool, but we can also leave it as is.